force-app/main/default/classes/ContactController.cls

```diff
@@ -50,4 +50,18 @@ public with sharing class ContactController {
         }
         update contactsForUpdate;
     }
+
+    @AuraEnabled
+    public static void updateContact(
+        Id recordId,
+        String firstName,
+        String lastName
+    ) {
+        Contact contact = new Contact(
+            Id = recordId,
+            FirstName = firstName,
+            LastName = lastName
+        );
+        update contact;
```

Comment:
```suggestion
        update as user contact;
```

Reply:
Dropping this for now as it is not included in prettier yet.